src/test/test_oop_analysis.py

```diff
@@ -5,26 +5,29 @@
 
 
 @pytest.mark.in_out
-@pytest.mark.parametetrize("first_row_first_val", "len_df", [(0.0, 101), (0.0, 101)])
+# @pytest.mark.parametetrize("first_row_first_val", "len_df", [(0.0, 2),
+# (0.0, 2)])
 # @pytest.mark.skip(reason="Already using unittest")
-def test_read_csv_as_df(len_df, first_row_first_val):
+def test_read_csv_as_df():
     """Test reading a csv file into a dataframe"""
 
     # given
-    file_in = r"J:\01_Projekte\SSC_Workshop\my-ssd-test-repo\data\expec.t"
-
+    file_in = (
+        "J:\\01_Projekte\\SSC_Workshop\\my-ssd-test-repo\\data\\test_data\\expec_test.t"
+    )
```

Comment:
@vivienzahs 
The pathstill absolute - can we change this to `../../data/test_data/expec_test.t`?

And you did not push the test file, did you? I cnnot find it in the remote.